### ISSUE TITLE
Create a build-requirements.txt for CentOS 8.

### ISSUE THREAD

**Issue by PaliC** [pytorch/multipy#253]
TL;DR: Figure out the necessary dependencies for CentOS 8, and record them.

`torch::deploy` is usually developed on using Ubuntu, however, aws instances are not the most accessible. Therefore, we need to better define the build instructions for OSs like CentOS 8.

- [ ] Clone the repository and download the submodules.
- [ ] Follow the instructions to [build `torch::deploy`](https://github.com/pytorch/multipy#installing-via-pip-install). Instead of doing the `pip install` at the end it may be easier to do the cmake build process described [here](https://github.com/pytorch/multipy#installing-via-pip-install). 
- [ ] You will definitely  run into problems when trying to install the packages in `build-requirements.txt`. However, it is a good starting point. Run `cp build-requirements.txt build-requirements-centos8.txt`, and replace the packages with the CentOS 8 equivalents. Then install the build requirements using `build-requirements-centos8.txt` and try to finish building `torch::deploy`. There will be a decent amount of trial and error.
- [ ] Modify the README.md to call out `build-requirements-centos8.txt`
- [ ] Change the name of `build-requirements.txt` -> `build-requirements-debian.txt`. Then replace all instances of `build-requirements.txt` with `build-requirements-debian.txt` in the repository.
- [ ] [Submit a PR](https://opensource.com/article/19/7/create-pull-request-github), click on the import button in the PR, and ask for a review.

Please contact Sahan Paliskara if you run into any problems.
